packages/ref-data-uploaders/txc-uploader/txc_processor.py

```diff
@@ -147,6 +147,8 @@ def collect_journey_pattern_section_refs_and_info(raw_journey_patterns):
 
 def safeget(dct, *keys):
     for key in keys:
+        if dct is None:
+            return None
         try:
             dct = dct[key]
         except KeyError:
```

Comment:
Maybe just make this `except:` without specifying the error type, then it will just return None in the case of any error

Reply:
done